### ISSUE TITLE
Winget Update

### ISSUE THREAD

**Issue by AndreM222** [neovim/neovim#26717]
### Problem

Currently I use winget for neovim but since version 0.9.2 no updates have been made

### Expected behavior

To be able to update to the latest stable version of nvim

**Comment by zeertzjq** [comment]
It seems that you can get the latest nightly version at winget.

**Comment by AndreM222** [comment]
That is true but not the stable version

**Comment by AndreM222** [comment]
The current stable version is 0.9.4

**Comment by MoaidHathot** [comment]
It seems the release Action has failed on `winget`
https://github.com/neovim/neovim/actions/runs/6461476354/job/17541555970

**Comment by AndreM222** [comment]
So a new solution is being worked out?

**Comment by dundargoc** [comment]
Not actively by us. I've tried to reach @vedantmgoyal2009 who set up our winget workflow to suggest how we should move forward.

**Comment by dundargoc** [comment]
@vedantmgoyal2009 I understand. I'm aware you've been blocked by external factors. To be clear, this is less to do with success/failure rate of the workflow and more about setting the correct expectations that neovim is not directly responsible for winget releases, as well as enforcing our (admittedly non-written) policies around not letting distribution-specific code unless necessary. We are very grateful for the work you've put in so far, and we would be delighted to continue to have neovim on winget if you have the time and energy to follow up.

We have created `https://github.com/neovim/neovim-releases` specifically for releases we would like but are hesitant to support officially and I think winget would be perfect there. Even though it's still under the neovim organization I believe the separation, as well as the README disclaimer should make it clear that it's provided on a best effort basis. I have created https://github.com/neovim/neovim-releases/pull/8 which I believe should be fully functional and equivalent to the workflow we had. I think everything is finished except for creating the necessary credential which I'm unsure about. If you want to take a look at it/help out that would be awesome!
